app/javascript/stylesheets/aplication.scss

```diff
@@ -1 +1,33 @@
-@import "~bootstrap";
\ No newline at end of file
+@import "~bootstrap";
+
+
+.flickr_pagination {
+  text-align: center;
+  padding: 0.3em;
+  cursor: default; }
+.flickr_pagination a, .flickr_pagination span, .flickr_pagination em {
+  padding: 0.2em 0.5em;
+  border-radius: 5px; }
+.flickr_pagination .disabled {
+  color: #aaaaaa; }
+.flickr_pagination .current {
+  font-style: normal;
+  font-weight: bold;
+  color: black; }
+.flickr_pagination a {
+  border: 1px solid #3d3434;
+  color: #3d3434;
+  text-decoration: none; }
+.flickr_pagination a:hover, .flickr_pagination a:focus {
+  border-color: gray;
+  background: #3d3434;
+  color: gray; }
+.flickr_pagination .page_info {
+  color: #aaaaaa;
+  padding-top: 0.8em; }
+.flickr_pagination .previous_page, .flickr_pagination .next_page {
+  border-width: 2px; }
+.flickr_pagination .previous_page {
+  margin-right: 1em; }
+.flickr_pagination .next_page {
+  margin-left: 1em; }
```

Comment:
use nested rules to make styles look more compact 
https://sass-scss.ru/documentation/rasshirenie_css/vlozhennie_pravila/
https://sass-scss.ru/documentation/rasshirenie_css/ssilka_na_roditelya_selektora/